src/main/java/core/basesyntax/FigureInformation.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureInformation {
```

Comment:
Don't use verbs for class/interface names. The name `FigureInformation` suggests an action rather than an interface. Consider renaming the interface to something that represents its purpose more clearly, like `Informative` or `InformationProvider`.